### ISSUE TITLE
error in "g_to_c()" mapping function

### ISSUE THREAD

**Issue by bitbub** [biocommons/hgvs#461]
I am running conversion variant mapper function "variant_mapper.g_to_c()" , and I got following error for genomic variant "NC_000002.11:g.73675227_73675228insCTC" and transcript "NM_015120.4". 

I don't fully understand error. Why it's generating for error for above specific annotation and not other genomic insertion variant.

"""
Traceback (most recent call last):
  File "annotated_vcf_with_HGVS.py", line 138, in <module>
    transcript_annotation = variant_mapper.g_to_c(genomic_variant, transcript)
  File "/Users/bonnypatel/miniconda3/envs/python2/lib/python2.7/site-packages/hgvs/assemblymapper.py", line 83, in g_to_c
    return self._maybe_normalize(var_out)
  File "/Users/bonnypatel/miniconda3/envs/python2/lib/python2.7/site-packages/hgvs/assemblymapper.py", line 189, in _maybe_normalize
    return self._norm.normalize(var)
  File "/Users/bonnypatel/miniconda3/envs/python2/lib/python2.7/site-packages/hgvs/normalizer.py", line 94, in normalize
    var = self.hm.c_to_n(var)
  File "/Users/bonnypatel/miniconda3/envs/python2/lib/python2.7/site-packages/hgvs/variantmapper.py", line 242, in c_to_n
    self._validator.validate(var_c)
  File "/Users/bonnypatel/miniconda3/envs/python2/lib/python2.7/site-packages/hgvs/validator.py", line 33, in validate
    return self._ivr.validate(var, strict) and self._evr.validate(var, strict)
  File "/Users/bonnypatel/miniconda3/envs/python2/lib/python2.7/site-packages/hgvs/validator.py", line 53, in validate
    raise HGVSInvalidVariantError(msg)
hgvs.exceptions.HGVSInvalidVariantError: insertion length must be 1
""""

**Comment by reece** [comment]
Projecting a variant that overlaps a sequence alignment isn't yet supported. Unfortunately, we don't even warn about it. This is one of those cases.

It's related to #259 and #353. 

A (poor) workaround for the moment is to use the VariantMapper, which doesn't validate. You'll get this:

```
$ hgvs-shell
>>> var_g = hp.parse_hgvs_variant("NC_000002.11:g.73675227_73675228insCTC")
>>> var_c = vm.g_to_c(var_g, "NM_015120.4")
>>> str(var_c)
'NM_015120.4:c.1573_1577insCTC'
```

Note that the 27_28 genomic insert expanded to 73_77. That's because the variant is right at the junction of a 3nt insertion. As described (kinda) in the other issues is that the behavior here is ill-defined.  

Here's the alignment: 
```
>>> print(variant_context_w_alignment(am37, var_g, tx_ac="NM_015120.4", margin=10))
   NM_015120.4  c       1563 >                            >     1587 NM_015120.4:c.1577_1579dup
   NM_015120.4  n       1674 > GGCTGTAAGTTCTCCTCTAGAAACT  >     1698 NM_015120.4:n.1688_1690dup
                               |||||||||||   |||||||||||  11=3I11= 
   NC_000002.11 g   73675217 > GGCTGTAAGTT---CTCTAGAAACT  > 73675238 NC_000002.11:g.73675227_73675228insCTC
   NC_000002.11 g   73675217 < CCGACATTCAA---GAGATCTTTGA  < 73675238 NC_000002.11:g.73675227_73675228insCTC
                                         ><                NC_000002.11:g.73675227_73675228insCTC
```

Although this seems simple because your insertion is exactly the difference between the two sequences, the general solution to handling variants in the context of alignment gaps is tricky (or, at least to me it is).

**Comment by reece** [comment]
See also #495

**Comment by icebert** [comment]
Resolved in hgvs release 1.2.0
